src/accordion.tsx

```diff
@@ -0,0 +1,346 @@
+        }
+        if (onChange) {
+            const currentItemValue = (isControlledByParent ? getValueAsList(value) : index).includes(item);
+            onChange(item, !currentItemValue);
+        }
+    };
+
+    if (isControlledByParent) {
+        return [getValueAsList(value), toggle];
+    }
+
+    return [index, toggle];
+};
+
+const getAccordionItemIndex = (element: Element | null) => {
+    const accordionAncestor = element?.closest('[data-accordion]');
+    if (!accordionAncestor) return undefined;
+
+    return Array.from(accordionAncestor.querySelectorAll('[data-accordion-item]'))
+        .filter((e) => e.closest('[data-accordion]') === accordionAncestor)
+        .findIndex((e) => e === element);
+};
+
+const AccordionItemContent = React.forwardRef<TouchableElement, AccordionItemContentProps>(
+    ({content, dataAttributes, trackingEvent, ...props}, ref) => {
+        const panelContainerRef = React.useRef<HTMLDivElement | null>(null);
+        const itemRef = React.useRef<HTMLDivElement | null>(null);
+        const {index, toogle} = useAccordionContext();
+        const isInverse = useIsInverseVariant();
+        const labelId = useAriaId();
+        const panelId = useAriaId();
+
+        const [itemIndex, setItemIndex] = React.useState<number>();
+        const isOpen = itemIndex !== undefined && index?.includes(itemIndex);
+
+        React.useEffect(() => {
+            setItemIndex(getAccordionItemIndex(itemRef.current));
+        }, []);
+
+        return (
+            <div ref={itemRef} {...getPrefixedDataAttributes({...dataAttributes, 'accordion-item': true})}>
+                <BaseTouchable
+                    ref={ref}
+                    className={classNames(styles.itemContent, {
+                        [styles.touchableBackground]: !isInverse,
+                        [styles.touchableBackgroundInverse]: isInverse,
+                    })}
+                    onPress={() => {
+                        if (itemIndex !== undefined) toogle(itemIndex);
+                    }}
+                    trackingEvent={trackingEvent}
+                    aria-expanded={isOpen}
+                    aria-controls={panelId}
+                >
+                    <Box paddingX={16}>
+                        <HeaderContent
+                            labelId={labelId}
+                            type="custom"
+                            {...props}
+                            right={
+                                <div className={styles.chevronContainer}>
+                                    <IconChevron
+                                        size={24}
+                                        className={isOpen ? styles.openChevron : styles.closeChevron}
+                                        color={
+                                            isInverse
+                                                ? skinVars.colors.inverse
+                                                : skinVars.colors.neutralMedium
```

Comment:
chevron changes to `neutralHigh` when the accordion is opened. Please, add transition

![image](https://github.com/Telefonica/mistica-web/assets/6722153/d45de6d3-056e-4c98-abdb-cd18de189b43)

Reply:
fixed